### PR TITLE
Persian language PR conflicts resolve #216

### PR DESCRIPTION
**Pull request by contributorpw**

### CODE REVIEW

**Comment by contributorpw** [comment]
@lydiahallie this PR automatically closes #216 too.

File: fa-IR/README-fa_IR.md [hunk 1]
```diff
@@ -0,0 +1,2745 @@
...
+- د: `undefined` و `ReferenceError`
+
+<details><summary><b>پاسخ</b></summary>
+<p>
+
+#### پاسخ: گزینه د
+
+Within the function, we first declare the `name` variable with the `var` keyword. This means that the variable gets hoisted (memory space is set up during the creation phase) with the default value of `undefined`, until we actually get to the line where we define the variable. We haven't defined the variable yet on the line where we try to log the `name` variable, so it still holds the value of `undefined`.
+
+Variables with the `let` keyword (and `const`) are hoisted, but unlike `var`, don't get <i>initialized</i>. They are not accessible before the line we declare (initialize) them. This is called the "temporal dead zone". When we try to access the variables before they are declared, JavaScript throws a `ReferenceError`.
```
Comment (jakeherp) [inline fa-IR/README-fa_IR.md:66]: All the explanations are still in English, these will need to be translated before this PR can be merged.

**Review by jakeherp — CHANGES_REQUESTED** [review]
All the explanations are still in English, these will need to be translated before this PR can be merged.

**Comment by jakeherp** [comment]
Closing due to inactivity

**Pull request closed without merging**